blog/models.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.conf import settings
+
+# Create your models here.
+class User(AbstractUser):
+    pass
```

Comment:
The User model is currently empty. If you don't need to extend the default User model, you can remove this model and use the default one provided by Django. However, if you plan to add custom fields in the future, you can leave it as it is.